hack/create-flakefinder-report.sh

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bash
+
+set -xeuo pipefail
+
+default_number_of_prs=20
+default_subdir_regex='pull-kubevirt-e2e-k8s-1\.[0-9]+-(sig-.*|operator)'
+default_state_of_prs='all'
+
+function usage {
+    cat <<EOF
+usage: $0 [-n number-of-prs] [-r subdir-regex] [-s state-of-prs]
```

Comment:
Did you consider using cobra subcommands to provide different entrypoints to the golang tool? I am just wondering why you are building another api on top of the tool in another language.